crates/re_types/definitions/rerun/components/position2d.fbs

```diff
@@ -13,7 +13,8 @@ namespace rerun.components;
 struct Position2D (
   "attr.python.aliases": "npt.NDArray[np.float32], Sequence[float], Tuple[float, float]",
   "attr.python.array_aliases": "npt.NDArray[np.float32], Sequence[float]",
-  "attr.rust.derive": "Copy, PartialEq, PartialOrd",
+  "attr.rust.derive": "Copy, PartialEq, PartialOrd, bytemuck::Pod, bytemuck::Zeroable",
```

Comment:
It's a bit weird to be comparing positions with `<`. `glam::Vec3` doesn't implement `PartialOrd`, for instance. I'm not sure it _hurts_ either, but perhaps it is a bit of a footgun?